src/ServiceControl.Audit/Auditing/AuditPersister.cs

```diff
@@ -260,12 +260,19 @@ async Task ProcessAuditMessage(MessageContext context)
                     enricher.Enrich(enricherContext);
                 }
 
+                var processingStartedTicks =
```

Comment:
This would be such a nice use case of [`String.Create`](https://docs.microsoft.com/en-us/dotnet/api/system.string.create?view=net-5.0) which is not available on .NET Framework :(